ipvsadm/Makefile.patch

```diff
@@ -0,0 +1,14 @@
+--- Makefile.original	2019-01-11 02:31:14.822002000 +0000
++++ Makefile	2019-01-11 02:33:14.604155000 +0000
+@@ -46,9 +46,9 @@
+ STATIC_LIBS	= libipvs/libipvs.a
+ 
+ ifeq "${ARCH}" "sparc64"
+-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow
++    override CFLAGS += -Wall -Wunused -Wstrict-prototypes -g -m64 -pipe -mcpu=ultrasparc -mcmodel=medlow $(LDFLAGS) -Wl,-R$(shell hab pkg path "core/popt")/lib -Wl,-R$(shell hab pkg path "rakops/libnl")/lib
+ else
+-    CFLAGS = -Wall -Wunused -Wstrict-prototypes -g
++    override CFLAGS += -Wall -Wunused -Wstrict-prototypes -g $(LDFLAGS) -Wl,-R$(shell hab pkg path "core/popt")/lib -Wl,-R$(shell hab pkg path "rakops/libnl")/lib
```

Comment:
This should probably be `core/libnl` 😄 

A pattern you might look at here is instead of shelling out to `hab pkg path` which will always get you latest, which _may_ not be what you want,  you can patch with the string `@@HAB_POPT_PATH@@` and then sed inline replace with the build helper `$(pkg_path_for core/popt)` to guarantee the path.  It will also give you a build error if you accidentally remove `core/popt` from your deps.